test/test-utils.ts

```diff
@@ -74,6 +74,8 @@ export type TestVars = {
   dataOwnerDetails: { [key: string]: UserDetails }
 }
 
+export type RecaptchaType = 'recaptcha' | 'friendly-captcha'
```

Comment:
Already created twice in classes above: Do not duplicate

Reply:
fixed